application/web/main.css

```diff
@@ -305,44 +307,6 @@ button.add-note:hover {
     justify-content: space-between;
 }
 
-.home-entry-descriptor {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    padding-right: 60px; 
-    padding-bottom: 10px;
-}
-
-.home-single-entry{
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    width: 100%;
-    padding: 10px 0px 2px; 
-    border-bottom: 2px solid var(--grey-text-color);
-}
-
-.home-single-entry-button {
-    font-size: 15px;
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    width: 100%;
-    background-color: var(--journal-entry);
-    border: none;
-    cursor: pointer;
-
-}
-
-.home-list img[src="../web/assets/vdots-entry-black.png"]{
-    background-color: var(--journal-entry);
-    border: none;
-    margin-left: 20px;
-    width: 22px;
-    height: 22px;
-    cursor: pointer;
-}
-
```

Comment:
Can you add this back to the proper location. I think you deleted it when you were merging with main